src/ceph-disk/ceph_disk/main.py

```diff
@@ -2362,17 +2362,18 @@ def create_key(self):
         cluster = self.args.cluster
         bootstrap = self.args.prepare_key_template.format(cluster=cluster,
                                                           statedir=STATEDIR)
-        command_check_call(
+        command_with_stdin(
```

Comment:
OK, I've added a tracker.

The thing with this commit is that it is *not* really a backport since 9fa13d4 actually removed this code in ceph-disk and replaced it with code which takes advantage of the 'osd new' command which is not available in jewel. So, whilst two of these commits are clearly backports, it's not clear that this commit is a backport, at least not to me at this stage.